mdsuite/__init__.py

```diff
@@ -27,13 +27,13 @@
 import logging
 import sys
 
-from .experiment import Experiment
-from .graph_modules import adjacency_matrix
-from .project import Project
-from .utils import config
-from .utils.report_computer_characteristics import Report
+from mdsuite.experiment import Experiment
+from mdsuite.project import Project
+from mdsuite.utils import config
+from mdsuite.utils.molecule import Molecule
+from mdsuite.utils.report_computer_characteristics import Report
 
-__all__ = ["Project", "Experiment", "adjacency_matrix", "Report", "config"]
+__all__ = ["Project", "Experiment", "Report", "config", "Molecule"]
```

Comment:
pls use ``Project.__name__ ``instead of ``"Project"``. The bare strings make automatic renaming impossible

Reply:
Sure, is that for everything in this list?

Reply:
yes for everything

Reply:
Mostly done in 84b4e87 however the config cannot be changed here.